docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -580,7 +596,8 @@ Use the following key points to find the right balance for securing your Quarkus
 The OIDC adapter uses cookies to keep the session, code flow, and post-logout state.
 This state is a key element controlling the lifetime of authentication data.
 
-Use the `quarkus.oidc.authentication.cookie-path` property to ensure that the same cookie is visible when you access protected resources with overlapping or different roots, for example:
+Use the `quarkus.oidc.authentication.cookie-path` property to ensure that the same cookie is visible when you access protected resources with overlapping or different roots.
```

Comment:
```suggestion
Use the `quarkus.oidc.authentication.cookie-path` property to ensure the same cookie is visible when accessing protected resources with overlapping or different roots.

```

Reply:
Propose to keep this one as is for clarity on who is accessing the protected resources.